### PR TITLE
Update Node and Yarn

### PR DESCRIPTION
**Pull request by jbalboni**
Updating us to reasonably recent version of yarn and Node.

### CODE REVIEW

**Comment by ncksllvn** [comment]
Does `.nvmrc` need to be updated as well?

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-6.11.1
+8.10.0
```
Comment (rianfowler) [inline .nvmrc]: `async await` 🎆

File: package.json [hunk 2]
```diff
@@ -211,6 +211,7 @@
     "modernizr": "^3.6.0",
     "modernizr-loader": "^1.0.1",
     "nyc": "^11.1.0",
+    "prop-types": "^15.6.1",
```
Comment (rianfowler) [inline package.json]: No more deprecation warning 👍

**Review by rianfowler — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (jbalboni) [reply package.json]: Eh, I think that's still there until we update some other dependencies.

**Review by ncksllvn — APPROVED** [review]
LGTM! This is exciting!

**Review by edpaget — APPROVED** [review]
:rocket:

**Pull request merged**